addons/markers/functions/fnc_removeTimestamp.sqf

```diff
@@ -0,0 +1,46 @@
+#include "script_component.hpp"
+/*
+ * Author: commy2
+ * Removes timestamp from end of a string.
+ *
+ * Arguments:
+ * 0: Marker text string <STRING>
+ *
+ * Return Value:
+ * Input with timestamp removed <STRING>
+ *
+ * Example:
+ * "abc [12:00 am]" call ace_markers_fnc_removeTimestamp // "abc"
+ * "[13:37]" call ace_markers_fnc_removeTimestamp // ""
+ * "xyz [123]" call ace_markers_fnc_removeTimestamp // "xyz [123]"
+ *
+ * Public: No
+ */
+#define DIGITS "0123456789"
+
+params ["_original"];
+
+// @todo, 2.02 reverse command will support STRING types
+private _string = toArray _original;
+reverse _string;
+_string = toString _string;
+
+if (_string select [0, 1] != "]") exitWith {_original};
+
+private _timestampLength = (_string find "[") + 1;
+_string = _string select [0, _timestampLength];
+
+// @todo
+_string = toArray _string;
+reverse _string;
+_string = toString _string;
+
+if (_string select [0, 1] != "[") exitWith {_original};
+
+if !(_string select [1, 1] in DIGITS) exitWith {_original};
+if !(_string select [2, 1] in DIGITS) exitWith {_original};
+if (_string select [3, 1] != ":") exitWith {_original};
```

Comment:
Fails if using `"HH"` format

Reply:
I think we just need to drop this and the next few lines to support time codes like `[12]`
or just drop HH format 
(I'm also not sure I understand why we have `"HH:MM:SS:MM"` but that's unrelated)

Reply:
@commy2 - I commited a fix to this PR, let me know if it looks ok